python/src/lib.rs

```diff
@@ -1,7 +1,9 @@
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
does this need to be disabled?

Reply:
Ah thanks. I had disabled earlier while refactoring and forgot to re-enable.